src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color color = Color.values()[new Random().nextInt(Color.values().length)];
+        return color.toString();
```

Comment:
Use `name()` instead of `toString()` to get the name of the enum constant as per the checklist.